docs/api.md

```diff
@@ -1083,6 +1085,65 @@ by the following errors
   Feature not enabled
 
 
+#### GET /account/device/messages
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-get-accountdevicemessages-->
+Retrieve stored messages for the current device.
+The query parameters will be passed as-is to the Pushbox service: it is
+your responsability to set `index` and `limit`.
+The response is echoed from the Pushbox service.
+<!--end-route-get-accountdevicemessages-->
+
+##### Query parameters
+
+* `service`: *string, valid('sendtab'), required*
```

Comment:
And if we *do* want to expose this abstraction, would it fit better as a path component like "/account/device/messages/sendtab" rather than a query parameter?

Sorry for the stream-of-conciousness, just thinking through how the API fits with the rest of the service here...